common/src/main/java/com/wynntils/models/gear/type/GearType.java

```diff
@@ -38,34 +39,39 @@ public enum GearType {
                     Items.CHAINMAIL_CHESTPLATE,
                     Items.IRON_CHESTPLATE,
                     Items.GOLDEN_CHESTPLATE,
-                    Items.DIAMOND_CHESTPLATE)),
+                    Items.DIAMOND_CHESTPLATE),
+            9),
     LEGGINGS(
             null,
             Items.LEATHER_LEGGINGS,
             0,
-            List.of(Items.CHAINMAIL_LEGGINGS, Items.IRON_LEGGINGS, Items.GOLDEN_LEGGINGS, Items.DIAMOND_LEGGINGS)),
+            List.of(Items.CHAINMAIL_LEGGINGS, Items.IRON_LEGGINGS, Items.GOLDEN_LEGGINGS, Items.DIAMOND_LEGGINGS),
+            10),
     BOOTS(
             null,
             Items.LEATHER_BOOTS,
             0,
-            List.of(Items.CHAINMAIL_BOOTS, Items.IRON_BOOTS, Items.GOLDEN_BOOTS, Items.DIAMOND_BOOTS)),
-    MASTERY_TOME(null, Items.ENCHANTED_BOOK, 0),
-    CHARM(null, Items.CLAY_BALL, 0);
+            List.of(Items.CHAINMAIL_BOOTS, Items.IRON_BOOTS, Items.GOLDEN_BOOTS, Items.DIAMOND_BOOTS),
+            11),
+    MASTERY_TOME(null, Items.ENCHANTED_BOOK, 0, -1),
+    CHARM(null, Items.CLAY_BALL, 0, -1);
 
     private final ClassType classReq;
     private final Item defaultItem;
     private final int defaultDamage;
     private final List<Item> otherItems;
+    private final int id;
```

Comment:
Maybe name this and related methods etc ´encodingId` to clarify?

Reply:
Only here, or every other enum?